utilities/virt.py

```
@@ -527,6 +534,18 @@ def to_dict(self):
                 if self.ssh_secret:
                     template_spec = self.update_vm_ssh_secret_configuration(template_spec=template_spec)
 
+        self._set_descheduler_exclusion()
+
+    def _set_descheduler_exclusion(self):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_set_descheduler_exclusion`

Add return type annotation: `None`

(ANN202)